vars/CondaPipeline.groovy

```diff
@@ -1,7 +1,33 @@
 import org.lsst.ts.jenkins.components.Csc
 
-def call(config_repo, name, module_name, arch="linux-64"){
+def call(Map pipeline_args= [:], @Deprecated config_repo=[], @Deprecated name="", @Deprecated module_name="", @ Deprecated arch="linux-64"){
     // Create a conda build pipeline
+    if(config_repo) {
+        echo 'config_repo positional parameter is deprecated use named parameter config_repo: [x] instead.'
+    }
+    if(name) {
+        echo 'name positional parameter is deprecated use named parameter name: x instead.'
+    }
+    if(module_name) {
+        echo 'module_name positional parameter is deprecated use named parameter module_name: x instead.'
+    }
+    if(arch) {
+        echo 'arch positional parameter is deprecated use named parameter arch: x instead.'
+    }
+    defaultArgs = [arch: "linux-64"]
+    pipeline_args = defaultArgs << pipeline_args
+    if(pipeline_args["config_repo"]){
+        config_repo = pipeline_args.config_repo
+    }
+    if(pipeline_args["name"]){
+        name = pipeline_args.name
+    }
+    if(pipeline_args["module_name"]){
+        module_name = pipeline_args.module_name
+    }
+    if(pipeline_args["arch"]){
```

Comment:
Wouldn't an `else {arch="linux-64"}` construction be better than what you did in line 18?